spec/browserstack_helper.rb

```diff
@@ -1,40 +1,40 @@
-# -*- coding: utf-8 -*-
```

Comment:
Personally, I'd like this to be more declarative.  I had to read it three times before I understood what it was meant to do.